### PR TITLE
add more Lambdaclass projects to starknet.toml

### PR DESCRIPTION
**Pull request by juanbono**

### CODE REVIEW

File: data/ecosystems/s/starknet.toml [hunk 1]
```diff
@@ -3838,17 +3838,11 @@ url = "https://github.com/l-henri/brain-dump"
 url = "https://github.com/l-henri/StarkNet-graffiti"
 
 [[repo]]
-url = "https://github.com/lambdaclass/cairo-rs"
-
-[[repo]]
-url = "https://github.com/lambdaclass/cairo-rs-py"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:3844]: please add https://github.com/lambdaclass/cairo-vm-py in favor of this deletion

File: data/ecosystems/s/starknet.toml [hunk 2]
```diff
@@ -3861,15 +3855,9 @@ url = "https://github.com/lambdaclass/circom_export_to_cairo"
 [[repo]]
 url = "https://github.com/lambdaclass/giza_example"
 
-[[repo]]
-url = "https://github.com/lambdaclass/kakarot"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:3865]: why remove?

File: data/ecosystems/s/starknet.toml [hunk 3]
```diff
@@ -3861,15 +3855,9 @@ url = "https://github.com/lambdaclass/circom_export_to_cairo"
...
-
 [[repo]]
 url = "https://github.com/lambdaclass/stark_compass_explorer"
 
-[[repo]]
-url = "https://github.com/lambdaclass/starknet-devnet"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:3871]: why remove?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a few comments

File: data/ecosystems/s/starknet.toml (hunk 2 above)
Comment (juanbono) [reply data/ecosystems/s/starknet.toml:3865]: it is just a fork

File: data/ecosystems/s/starknet.toml (hunk 3 above)
Comment (juanbono) [reply data/ecosystems/s/starknet.toml:3871]: same as before, it is a fork.

File: data/ecosystems/s/starknet.toml (hunk 1 above)
Comment (juanbono) [reply data/ecosystems/s/starknet.toml:3844]: https://github.com/lambdaclass/cairo-vm-py is already in the list (below at line 3847)

**Review by eherrerosj — CHANGES_REQUESTED** [review]
please keep the forks, we have fork-detection and handling algorithms in the ETL

File: data/ecosystems/s/starknet.toml (hunk 2 above)
Comment (juanbono) [reply data/ecosystems/s/starknet.toml]: added again :)

File: data/ecosystems/s/starknet.toml (hunk 3 above)
Comment (juanbono) [reply data/ecosystems/s/starknet.toml]: added again :)

**Comment by juanbono** [comment]
@eherrerosj I re-added the forks.

**Pull request merged**